roles/common/tasks/main.yml

```diff
@@ -44,16 +44,32 @@
   pip: name=pip version=18.1 state=present
   tags: common
 
+- name: ius repo
+  yum_repository:
+    name: ius
+    description: IUS Community Packages for Enterprise Linux {{
+      ansible_distribution_major_version }} - $basearch
+    mirrorlist: https://mirrors.iuscommunity.org/mirrorlist?repo=ius-centos{{
+      ansible_distribution_major_version }}&arch=$basearch&protocol=http
+    gpgkey: https://dl.iuscommunity.org/pub/ius/IUS-COMMUNITY-GPG-KEY
+    enabled: no
+  tags: common
+
+- name: install modern git version from ius
+  yum: name=git2u enablerepo=ius state=present
+  tags: common
+
 - name: uninstall python-requests
   yum: name=python-requests state=absent
   tags: common
 
 - name: configure sysadmin accounts
   user:
     name: "{{ item }}"
+    password: "{{ default_sysadmin_passwd }}"
     groups: wheel
+    append: yes
```

Comment:
@vtepliuk Check out how this option relates to the other in user module: https://docs.ansible.com/ansible/latest/modules/user_module.html